src/dewey/config.clj

```diff
@@ -12,25 +12,10 @@
   [props config-valid configs]
   "dewey.environment-name" "docker-compose")
 
-(cc/defprop-optstr amqp-host
-  "The hostname for the AMQP server"
+(cc/defprop-optstr amqp-uri
+  "The URI for the AMQP server"
   [props config-valid configs]
-  "dewey.amqp.host" "rabbit")
-
-(cc/defprop-optint amqp-port
-  "The port number for the AMQP server"
-  [props config-valid configs]
-  "dewey.amqp.port" 5672)
-
-(cc/defprop-optstr amqp-user
-  "The username for the AMQP server"
-  [props config-valid configs]
-  "dewey.amqp.user" "guest")
-
-(cc/defprop-optstr amqp-pass
-  "The password for the AMQP user"
-  [props config-valid configs]
-  "dewey.amqp.password" "guest")
+  "dewey.amqp.uri" "amqp://guest:guestPW@localhost:5672")
```

Comment:
This is a different default than before (should be `guest:guest` for username:password and `rabbit` rather than `localhost`, which is the default because that's what rabbitmq provides in a default "dev" setup in a docker container named `rabbit`, IIRC)